Split time zone info if it is given

meggart opened the pull request:
I just stumbled over this time unit string: `"seconds since 1970-01-01T00:00:00+00:00"` which threw an error because of the additional time zone information. This branch fixes the bug for now by simply throwing away the time zone information. Do you think it is ok to just ignore this? Maybe it would make sense to show a warning whenever the time shift is not equal to zero.

Alexander-Barth commented:
Thanks you for your PR!

> Maybe it would make sense to show a warning whenever the time shift is not equal to zero.

Yes, I agree. Or even an error?

I will add a note to the documentation that time zones are not supported.

meggart commented:
I personally would prefer a warning over an error. It woud be quite frustrating if you can not access the time info of a file at all. This way the user can still access and convert the data but just has to adjust time zones if applicable. I added some unit tests and removed the WIP from the title.

Alexander-Barth commented:
OK, thank you very much, Fabian. I will make a new release soon.
